pepper-apis/dss-database/src/main/resources/db-changes/schema/DDP-7248.xml

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                   https://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+    <changeSet author="Dmitrii Eliseev" id="20220427-audit-table">
+        <createTable tableName="audit_trail">
+            <column name="audit_trail_id" type="bigint" autoIncrement="true">
+                <constraints primaryKey="true" primaryKeyName="audit_trail_pk"/>
+            </column>
+
+            <column name="study_id" type="bigint" />
```

Comment:
Do you mean to make this Not null ? 
We need to set nullable="false" , default is true right ?

Reply:
What is your suggestion for the **onDelete** trigger of FK:
```
        <addForeignKeyConstraint baseTableName="audit_trail" baseColumnNames="study_id"
                                 referencedTableName="umbrella_study" referencedColumnNames="umbrella_study_id"
                                 constraintName="audit_trail_study_id_fk" onDelete="SET NULL"/>
```

Currently, study_id will be set to null. If we set the constraint to not null, then on delete of study we have to perform CASCADE DELETE of the audit log. But I'm not sure we want to delete the log.

Reply:
I don't believe we delete studies, we invalidate them using StudyBuilder --invalidate option

Reply:
So, then I will update this trigger to `ON DELETE RESTRICT` right? Is that what you suggest @ssettipalli ?

Reply:
I was thinking may be make the column study_id NOT NULL and remove onDelete="SET NULL" because we never delete complete study and any log entry is related to some study.
This is with assumption that we have log entries associated with some study for sure.

Reply:
Ok, will update the PR in a couple of minutes.